Berksfile

```diff
@@ -3,6 +3,6 @@ source "https://supermarket.getchef.com"
 metadata
 
 group :integration do
-  cookbook 'selinux', '~> 0.9.0'
+  cookbook 'selinux', git: 'https://github.com/jbartko/selinux.git', branch: 'feature/add-serverspec'
```

Comment:
I don't want to break things for you.
If our testing passes without the modified Berks, great.
If not, can we consider creating a special recipe in the test cookbook for
installing the SELinux package, and calling it instead of the regular one?

PS
Happy to hear someone is using my cookbook! :)

*Nitzan Raz*
http://backslasher.net

On Wed, Jan 27, 2016 at 8:33 PM, John Bartko <notifications@github.com>
wrote:

> In Berksfile
> <https://github.com/BackSlasher/chef-selinuxpolicy/pull/37#discussion_r51027564>
> :
>
> > @@ -3,6 +3,6 @@ source "https://supermarket.getchef.com"
> >  metadata
> >
> >  group :integration do
> > -  cookbook 'selinux', '~> 0.9.0'
> > +  cookbook 'selinux', git: 'https://github.com/jbartko/selinux.git', branch: 'feature/add-serverspec'
>
> The dependent selinux cookbook has broken Fedora support. The git branch
> corresponds to PR skottler/selinux#28
> <https://github.com/skottler/selinux/pull/28> which adds serverspec tests
> and updates Fedora support.
>
> If maintaining the dependency is valuable for the purpose of integration
> testing this cookbook, could you consider commenting on the PR? I'm not
> sure how attentive the maintainer is and wanted to give them an opportunity
> to review and merge changes instead of creating yet another fork.
>
> In the meantime, I will attempt to update this PR without git refs in the
> Berksfile.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/BackSlasher/chef-selinuxpolicy/pull/37/files#r51027564>
> .
>